tests/test_routes.py

```diff
@@ -183,4 +187,23 @@ def test_content_type_error(self):
         )
         self.assertEqual(resp.status_code, status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
 
+######################################################################
+#  PRODUCT ROUTE TEST CASES
+######################################################################
+    def test_create_product(self):
+        """ Create a new Product """
+        test_product = self._create_product()
+        test_product.create()
+        logging.debug(test_product)
+        resp = self.app.post(
+            "/products", json=test_product.serialize(), content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
+        # # Make sure location header is set
+        # location = resp.headers.get("Location", None)
+        # self.assertIsNotNone(location)
 
+        # # Check the data is correct
```

Comment:
Hey Catherine you forgot to uncomment your code JEEZE!

Reply:
I fixed it!